withdrawn-packages.txt

```diff
@@ -1,295 +1,3 @@
-mesa-libd3dadapter9-24.1.0-r0.apk
-mesa-libd3dadapter9-24.1.1-r0.apk
-mesa-libd3dadapter9-24.1.2-r0.apk
-mesa-libd3dadapter9-24.1.3-r0.apk
-mesa-libd3dadapter9-24.1.4-r0.apk
-mesa-libd3dadapter9-24.1.5-r0.apk
-mesa-libd3dadapter9-24.2.0-r0.apk
-mesa-libd3dadapter9-24.2.1-r0.apk
-mesa-libd3dadapter9-24.2.1-r1.apk
-mesa-libd3dadapter9-24.2.2-r0.apk
-mesa-libd3dadapter9-24.2.3-r0.apk
-mesa-libd3dadapter9-24.2.3-r1.apk
-mesa-libd3dadapter9-24.2.4-r0.apk
-mesa-libd3dadapter9-24.2.4-r1.apk
-mesa-libd3dadapter9-24.2.5-r0.apk
-mesa-libd3dadapter9-24.2.5-r1.apk
-mesa-libd3dadapter9-24.2.6-r0.apk
-mesa-libd3dadapter9-24.2.6-r1.apk
-mesa-libgallium-24.2.3-r1.apk
-mesa-libgallium-24.2.4-r0.apk
-mesa-libgallium-24.2.4-r1.apk
-mesa-libgallium-24.2.5-r0.apk
-mesa-libgallium-24.2.5-r1.apk
-mesa-libgallium-24.2.6-r0.apk
-mesa-libgallium-24.2.6-r1.apk
-mesa-osmesa-23.1.2-r0.apk
-mesa-osmesa-23.1.3-r0.apk
-mesa-osmesa-23.1.4-r0.apk
-mesa-osmesa-23.1.4-r1.apk
-mesa-osmesa-23.3.0-r0.apk
-mesa-osmesa-23.3.1-r0.apk
-mesa-osmesa-23.3.2-r0.apk
-mesa-osmesa-23.3.3-r0.apk
-mesa-osmesa-23.3.4-r0.apk
-mesa-osmesa-24.0.0-r0.apk
-mesa-osmesa-24.0.1-r0.apk
-mesa-osmesa-24.0.2-r0.apk
-mesa-osmesa-24.0.3-r0.apk
-mesa-osmesa-24.0.4-r0.apk
-mesa-osmesa-24.0.5-r0.apk
-mesa-osmesa-24.0.6-r0.apk
-mesa-osmesa-24.0.7-r0.apk
-mesa-osmesa-24.0.8-r0.apk
-mesa-osmesa-24.1.0-r0.apk
-mesa-osmesa-24.1.1-r0.apk
-mesa-osmesa-24.1.2-r0.apk
-mesa-osmesa-24.1.3-r0.apk
-mesa-osmesa-24.1.4-r0.apk
-mesa-osmesa-24.1.5-r0.apk
-mesa-osmesa-24.2.0-r0.apk
-mesa-osmesa-24.2.1-r0.apk
-mesa-osmesa-24.2.1-r1.apk
-mesa-osmesa-24.2.2-r0.apk
-mesa-osmesa-24.2.3-r0.apk
-mesa-osmesa-24.2.3-r1.apk
-mesa-osmesa-24.2.4-r0.apk
-mesa-osmesa-24.2.4-r1.apk
-mesa-osmesa-24.2.5-r0.apk
-mesa-osmesa-24.2.5-r1.apk
-mesa-osmesa-24.2.6-r0.apk
-mesa-osmesa-24.2.6-r1.apk
-mesa-xatracker-23.1.2-r0.apk
-mesa-xatracker-23.1.3-r0.apk
-mesa-xatracker-23.1.4-r0.apk
-mesa-xatracker-23.1.4-r1.apk
-mesa-xatracker-23.3.0-r0.apk
-mesa-xatracker-23.3.1-r0.apk
-mesa-xatracker-23.3.2-r0.apk
-mesa-xatracker-23.3.3-r0.apk
-mesa-xatracker-23.3.4-r0.apk
-mesa-xatracker-24.0.0-r0.apk
-mesa-xatracker-24.0.1-r0.apk
-mesa-xatracker-24.0.2-r0.apk
-mesa-xatracker-24.0.3-r0.apk
-mesa-xatracker-24.0.4-r0.apk
-mesa-xatracker-24.0.5-r0.apk
-mesa-xatracker-24.0.6-r0.apk
-mesa-xatracker-24.0.7-r0.apk
-mesa-xatracker-24.0.8-r0.apk
-mesa-xatracker-24.1.0-r0.apk
-mesa-xatracker-24.1.1-r0.apk
-mesa-xatracker-24.1.2-r0.apk
-mesa-xatracker-24.1.3-r0.apk
-mesa-xatracker-24.1.4-r0.apk
-mesa-xatracker-24.1.5-r0.apk
-mesa-xatracker-24.2.0-r0.apk
-mesa-xatracker-24.2.1-r0.apk
-mesa-xatracker-24.2.1-r1.apk
-mesa-xatracker-24.2.2-r0.apk
-mesa-xatracker-24.2.3-r0.apk
-mesa-xatracker-24.2.3-r1.apk
-mesa-xatracker-24.2.4-r0.apk
-mesa-xatracker-24.2.4-r1.apk
-mesa-xatracker-24.2.5-r0.apk
-mesa-xatracker-24.2.5-r1.apk
-mesa-xatracker-24.2.6-r0.apk
-mesa-xatracker-24.2.6-r1.apk
+redpanda-25.1-dev-25.2.1-r2.apk
+redpanda-25.1-compat-25.2.1-r2.apk
+redpanda-25.1-25.2.1-r2.apk
```

Comment:
There are also `-r1` and `-r0` packages to withdraw: https://apk.chaindag.dev/https/packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz@etag:3236664eb1090d72220f88756555f927/APKINDEX?depend=&full=true&provide=&search=redpanda-25.1